prr/__main__.py

```
@@ -89,22 +97,33 @@ def add_common_args(_parser):
             action="store_true",
             default=False,
         )
+
         _parser.add_argument("prompt_path", help="Path to prompt to run")
 
     add_common_args(run_parser)
     add_common_args(watch_parser)
     add_common_args(script_parser)
 
+    ui_parser.add_argument("prompt_path", help="Path to prompt to analyze")
+
     watch_parser.add_argument(
         "--cooldown", "-c", type=int, help="How much to wait after a re-run", default=5
     )
 
     args, prompt_args = parser.parse_known_args()
     parsed_args = vars(args)
 
+    if parsed_args["command"] == "ui":
+        command = UIPromptCommand(parsed_args)
+        command.start()
+
+    if not check_if_prompt_exists(parsed_args["prompt_path"]):
```

if there's no file provided as an argument it just crashes
ideally, we want to raise an error and show available flags